src/jit/emitxarch.cpp

```diff
@@ -5359,192 +5386,276 @@ void emitter::emitIns_AX_R(instruction ins, emitAttr attr, regNumber ireg, regNu
 }
 
 #ifdef FEATURE_HW_INTRINSICS
-void emitter::emitIns_SIMD_R_R_A(instruction ins, emitAttr attr, regNumber reg, regNumber reg1, GenTreeIndir* indir)
+void emitter::emitIns_SIMD_R_R_I(instruction ins, emitAttr attr, regNumber targetReg, regNumber op1Reg, int ival)
```

Comment:
This needs a function header.

Reply:
(And if you were inspired to, you could create headers for the pre-existing methods that don't have them).

Reply:
Fixed.